config/verificationMail.js

```diff
@@ -63,7 +63,7 @@ const config = {
     const template = Handlebars.compile(source);
     const { username, email } = params;
     const tokenToSend = await jwt.sign({ email }, process.env.SECRET);
-    const url = `${process.env.SERVER_ADDRESS}/api/v1/update_password/${tokenToSend}`;
+    const url = `${process.env.FRONT_END_URL}/update_password?token=${tokenToSend}`;
```

Comment:
Good work @shemaeric for the initiative. it should be better if you change this to `......updatePassword/${tokenToSend}` to be able to read for us to be able to read the token from the address bar on the frontend.

Reply:
That's true @niyobobo I was going to change it and thanks for reminding.